index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=1024px, initial-scale=1.0">
+  <title>Valentina Krivelskaya</title>
+</head>
+<body>
+  <header>
+    <h1>Valentina Krivelskaya</h1>
+    <address>
+      <div>Email: v.krivelskaya@mail.ru</div>
+      <div>Address: 54-234 Belinskogo str, Minsk, Belarus</div> 
+      <div>Phone: (29) 543 38 93</div>      
+       </address>    
+  </header>
+  
+</body>
+<img src="image.jpg" width="100px" alt="There are only tags in my mind">
+<main>
+  <section>
+    <h4>SUMMARY</h4>
+      <p>I have no experience in the field of IT, but I have a passion and perseverance to learn and become a professional in Front-End Development</p>
+  </section>
+  <section>
+    <h4>EXPERIENCE</h4>
+    <ul>
+      <li>Chief of Branch Management Accounting and Treasury Branch<br>
```

Comment:
br tag isn't a good practice. In your case better to use additional p tag or just li and add styles for width regulation. We can discuss it separately.